c2rust-refactor/src/scripting/utils.rs

```diff
@@ -0,0 +1,16 @@
+use rlua::prelude::{LuaContext, LuaResult, LuaTable, ToLua};
+
+pub(crate) fn iter_to_lua_array<'lua, I, T: 'lua>(iter: I, lua_ctx: LuaContext<'lua>) -> LuaResult<LuaTable<'lua>>
```

Comment:
Isn't this basically https://docs.rs/rlua/0.16.3/rlua/struct.Context.html#method.create_sequence_from?

Reply:
Nice find! Didn't see this.

Reply:
I wonder why my version is much shorter and straightforward than their impl though. huh.

Reply:
Their's is probably a lot faster.

Reply:
Goood point